src/Microsoft.Diagnostics.DebugServices/ISymbolService.cs

```diff
@@ -53,10 +58,11 @@ public virtual void Dispose()
         /// <param name="msdl">if true, use the public Microsoft server</param>
         /// <param name="symweb">if true, use symweb internal server and protocol (file.ptr)</param>
         /// <param name="symbolServerPath">symbol server url (optional)</param>
-        /// <param name="authToken"></param>
+        /// <param name="authToken">PAT for secure symbol server (optional)</param>
         /// <param name="timeoutInMinutes">symbol server timeout in minutes (optional)</param>
+        /// <param name="retryCount">number of retries (optional)</param>
         /// <returns>if false, failure</returns>
-        bool AddSymbolServer(bool msdl, bool symweb, string symbolServerPath, string authToken, int timeoutInMinutes);
```

Comment:
I am fine with it as it's not a publicly consumed API - but this changes public surface. We need to be careful about these as this goes public.

Reply:
I consider the public API still in flux.  We haven't locked it down yet.  If someone is taking a dependency on M.D.DebugServices outside of the repo, they will have to deal with these kinds of changes for now.